src/components/__stories__/artwork_filter.tsx

```diff
@@ -135,10 +136,23 @@ function FilterArtworksTotalCountExample() {
   )
 }
 
+function FilterArtworksExample() {
+  Relay.injectNetworkLayer(artsyNetworkLayer())
```

Comment:
I think there was a reason to have to call it each time, but can’t recall from the top of my head.